hw/ip/kmac/doc/_index.md

```diff
@@ -45,8 +45,8 @@ It is expected to protect against 1st order SCA attacks by implementing masked s
 
 The above figure shows the KMAC/SHA3 HWIP block diagram.
 The KMAC has register interfaces for SW to configure the module, initiate the hashing process, and acquire the result digest from the STATE memory region.
-It also has interfaces tot he KeyMgr to get the secret key (masked) and the message data.
-The interface to the KeyMgr is mainly for Key Derivative Functions (KDF).
+It also has an interface to the KeyMgr to get the secret key (masked).
+The IP has N x application interfaces, which allows other HWIPs to request any pre-defined hashing operations.
```

Comment:
small nit: maybe just link to the Application Interface section of the spec here, since all the further details are there?